pycbc/frame/__init__.py

```diff
@@ -2,6 +2,8 @@
                      query_and_read_frame, frame_paths, write_frame,
                      DataBuffer, StatusBuffer)
 
+from . store import (read_store)
```

Comment:
Isn't that space not supposed to be there?

Reply:
`from .store import read_store`?

Reply:
Either works. I'm not sure which is the preferred style.... Maybe .store is more common?